.github/workflows/yarn_test.yaml

```diff
@@ -0,0 +1,27 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [20.x]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: install yarn app
+      run: |
+        cd apps/nowcasting-app
+        yarn install
+    - name: run tests
```

Comment:
@braddf might have a response to this, but will this run the tests from the `test` script command in the `package.json`?